learning_wavelets/training_scripts/learnlet_training.py

```diff
@@ -147,21 +187,43 @@ def l_rate_schedule(epoch):
     norm_cback.on_train_batch_end = norm_cback.on_batch_end
 
 
-    n_channels = 1
-    # run distributed
+   # run distributed
     mirrored_strategy = tf.distribute.MirroredStrategy()
     with mirrored_strategy.scope():
-        model = learnlet(input_size=(None, None, n_channels), lr=1e-3, **run_params)
-    print(model.summary(line_length=114))
-
+        model = Learnlet(**run_params)
+        model.compile(
+            optimizer=Adam(lr=1e-3),
+            loss='mse',
+            metrics=[keras_psnr, keras_ssim],
+        )
+        inputs = [tf.zeros((1, 32, 32, 1)), tf.zeros((1, 1))]
+        model(inputs)
+
+    shearlets, _ = cl.get_shearlets(512, 512, 6)
```

Comment:
Why `6` ?

Reply:
Put the answer in comment if it makes sense

Reply:
fixed